documentation/guides/record-tests/test-actions/assertions.md

```diff
@@ -70,7 +70,7 @@ You can add the following options to assertions:
 
     To add this option, click the **Add Timeout** button in the assertion parameters area.
 
-    The *Assertion Timeout* options in the [Record Configuration](../../../user-interface/record-configuration-dialog.md) and [Run Configruations](../../../user-interface/run-configurations-dialog.md##advanced-options) dialogs specify the default timeout.
+    If you do not add this option, the default timeout is used. You can specify it by the *Assertion Timeout* options in the [Record Configuration](../../../user-interface/record-configuration-dialog.md) and [Run Configruations](../../../user-interface/run-configurations-dialog.md##advanced-options) dialogs.
```

Comment:
"specify it by the *Assertion Timeout* options" 'specify it by the options' is not correct. 

The default timeout is used if you do not add this option. To specify this option, use the *Assertion Timeout* options in the [Record Configuration] and [Run Configruations].

Reply:
Can we write: "If you do not add this option, the default timeout is used. You can specify it using the *Assertion Timeout* options in the [Record Configuration](../../../user-interface/record-configuration-dialog.md) and [Run Configruations](../../../user-interface/run-configurations-dialog.md##advanced-options) dialogs." ?

Reply:
or we can use your variant with little correction:
"The default timeout is used if you do not add this option. To specify the default timeout, use the Assertion Timeout options in the [Record Configuration] and [Run Configruations]."

And this variant is almost the same as I write before: "The default timeout is specified using the Assertion Timeout options in the ... dialogs."

Reply:
"The default timeout is used if you do not add this option. To specify the default timeout, use the Assertion Timeout options in the [Record Configuration] and [Run Configruations]." This variant is better because no '-ing'.

Reply:
Ok, the last variant is ok